fixtures/generic/color-temperature-fader.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Color Temperature Fader",
+  "categories": ["Other"],
+  "meta": {
+    "authors": ["Ken"],
+    "createDate": "2024-05-04",
+    "lastModifyDate": "2024-05-04"
+  },
+  "comment": "Generic fader + color temperature",
+  "links": {
+    "other": [
+      "http://example.com"
```

Comment:
I didn't see a way to avoid adding a URL, and it seems like the schema requires it, but the other generic fixtures don't have any?

Reply:
The schema actually does not require any links, but I made them mandatory in the fixture editor because people did often not add any links, and fixtures without links are really hard to review.